VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -15,20 +24,41 @@
     private DiscussionTextUtils() {
     }
 
-    public static CharSequence getAuthorAttributionText(@NonNull final IAuthorData authorData, @NonNull final Resources resources) {
-        final CharSequence formattedTime = DateUtils.getRelativeTimeSpanString(
-                authorData.getCreatedAt().getTime(),
-                System.currentTimeMillis(),
-                DateUtils.MINUTE_IN_MILLIS,
-                DateUtils.FORMAT_NUMERIC_DATE|DateUtils.FORMAT_SHOW_YEAR);
+    public static void setAuthorAttributionText(@NonNull TextView textView, @NonNull final IAuthorData authorData, @NonNull final Router router) {
```

Comment:
Note that you could have the `Router` injected statically here if you add it to the list in the `requestStaticInjection()` call from `EdxDefaultModule`. Just an option to consider if you want to eliminate injected parameters.